components/Nav/Nav.tsx

```diff
@@ -213,18 +217,19 @@ const Nav = () => {
                 </Link>
                 {/* Mobile menu button */}
                 <Disclosure.Button className="inline-flex items-center justify-center p-2 rounded-md text-neutral-400 hover:text-white hover:bg-neutral-900 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white">
-                  <span className="sr-only">Open main menu</span>
+                  {/* <span className="sr-only">Open main menu</span>
                   {open ? (
```

Comment:
Usual comment of look over for any commented code and throw it away 😉

Reply:
I will remove all commented code now and checkout the build :)